scripts/get_version.py

```diff
@@ -2,52 +2,72 @@
 from VSTSRequest import APIRequest
 
 def get_builds(response):
-    print(f'status code: {response.status_code}')
+    print('status code: {r.status_code}'.format(r=response))
     return [str(build['id']) for build in response.json()['value']]
 
-def main(account, project, definition, branch, status, total, version, user, token):
+def main(
+        account,
+        project,
+        definition,
+        branch,
+        buildstatus,
+        count,
+        version,
+        user,
+        token,
+        *args,
+        **kwargs):
     import os
-    req = APIRequest(version, account, project, definition, branch, status, total, user, auth=token)
-
-    print(f'Sending request to: {req.url}')
-    print(f'Request Parameters: {req.params}')
 
+    req = APIRequest(version, account, project, definition, branch, buildstatus, count, user, token)
```

Comment:
If we ran this as a build def in vsts would it have to be on msdata so it could get a token that could access it?

Reply:
I am running it on msasg and using a token for the msdata account. If we use the oauth token that is generated for us during the execution (System.AccessToken) we do need to run it in msdata.